tools/integration/src/index.ts

```diff
@@ -271,11 +287,144 @@ yargs
                 demandOption: true
             });
     }, handlePublish)
+	.command('lint', 'provides linting for package', (yargs) => {
+    	return yargs
+        	.option('path', {
+        		alias: 'p',
+                describe: 'path of the package',
+                type: 'string',
+                demandOption: false
+            })
+    }, handleLint)
     .demandCommand(1, 'You need at least one command before moving on')
     .help()
     .alias('help', 'h')
     .argv;
 
+// Function to handle lint logic
+async function handleLint(argv: any) {
+    const currentDirectory = process.cwd();
+    console.log(`Current working directory: ${currentDirectory}`);
```

Comment:
Probably we shall use logger instead of console consistently throughout the lint function.